js/src/behaviors/dismiss.ts

```diff
@@ -0,0 +1,46 @@
+export function initDismiss() {
+	window.addEventListener(
+		'click',
+		({ target }: Event) => {
+			const dismissButton = target instanceof Element && target.closest('[data-dismiss]');
+
+			if (!dismissButton) {
+				return;
+			}
+
+			const dismissTarget = dismissButton.closest('[data-dismissable]');
+
+			if (!dismissTarget) {
+				return;
+			}
+
+			dismissElement(dismissTarget);
+		},
+		{ passive: true }
```

Comment:
I think `passive` is mostly meant for scroll events or similar. I'm not sure it provides any benefits for click events, though I could be wrong on that front.

Reply:
You're right, it's not needed here. Updated